src/main/java/com/iexec/common/chain/validation/EthereumNonZeroAddressValidator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
```

Comment:
I don't remember the decision on copyright year.
Do we keep homogeneous values or real year values ?

Reply:
Fixed copyright thanks!

Copyright dates are tighten to release dates